.github/workflows/push.yml

```diff
@@ -77,7 +77,9 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: ghcr.io/${{ github.repository }}:file-storage-${{ needs.branch_meta.outputs.sha }}
-          labels: ${{ steps.docker_meta_img.outputs.labels }}
+          labels: |
+            ${{ steps.docker_meta_img.outputs.labels }}
+            org.opencontainers.image.title=schulcloud-file-storage
```

Comment:
Dosn't wrok like it shoud be.
The command has now two times the org.opencontainers.image.title as agrument.
https://github.com/hpi-schul-cloud/schulcloud-server/actions/runs/5795438880/job/15707019590#step:8:138